### PR TITLE
Add readme and link quizcategory pages

### PR DESCRIPTION
**Pull request by Maianki**
## Added README and linked question page to other categories in Quiz
Screenshot (615)

## Check the README and site's live link here
[Click here to see README live ](https://github.com/Maianki/Shuttle-Quiz-app/tree/add-README-and-link-quizcategory-pages)
[Site live link](https://ecstatic-kowalevski-58af49.netlify.app/)

### CODE REVIEW

File: quiz-pages/badminton-trivia-quiz/rules.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+               
+                <div class="card-body align-items-center">
+                    <ul>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>There will be 5 quesitons.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Each question will have one correct answer.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Correct answer will provide 5 points.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>Incorrect answer will deduct 1 point.</li>
+                        <li class="list-unstyled quiz-rules text-bold-500"><i class="md-ht-1 fas fa-pencil-alt"></i>To unlock new level you have to score 75%.</li>
+                        
+                        <li class="list-unstyled"></li>
```
Comment (partha8) [inline quiz-pages/badminton-trivia-quiz/rules.html]: Is this required here? The last li tag

**Review by partha8 — APPROVED** [review]
Looks good, clean code and good class names. Good job!

File: quiz-pages/badminton-trivia-quiz/rules.html (hunk 1 above)
Comment (Maianki) [reply quiz-pages/badminton-trivia-quiz/rules.html]: No, forgot to remove it. Thanks for pointing this out. Will fix this.

**Review by sainath7878 — APPROVED** [review]
Looks good.You've written neat and clean code. Great work!

**Comment by Maianki** [comment]
> Looks good. You've written neat and clean code. Great work!

Thanks for reviewing.

**Comment by Maianki** [comment]
Done the said changes. Merging PR.

**Pull request merged**